charts/castai-kvisor/templates/rbac.yaml

```diff
@@ -80,6 +80,23 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
```

Comment:
Would be good to have something like and add conditional.
```yaml
policyEnforcement:
    enabled: false
```

We don't want to impact current installed users with such new feature.